utilities/bids_loaddesign.m

```diff
@@ -0,0 +1,94 @@
+function [design, stimdur, ints2unique, unique2ints] = bids_loaddesign(sub,BIDS_path,session_type, session_number, tr)
+%%% 2020-05-30 written by Alex Kipnis (tested on https://openneuro.org/datasets/ds001246/versions/1.2.1 after running fMRIprep on sub-01) 
+%     Loads event markers (tsv files) into "design" cell-array, which is used as
+%     input for GLMdenoise. This script assumes that the presentation
+%     duration for each stimulus is constant and regressors that are not
+%     stimulus IDs (e.g. prespecified noise regressors) will be added to the design matrix separately. 
+%
+%   
+%     Parameters: 
+%     sub (int): subject number
+%     BIDS_path (str): path to BIDS-formatted folder 
+%     session_type (str): all folders containing this substring will be
+%       searched for .tsv files
+%     session_number (int): Self-explanatory (GLMs should be fit with
+%       sessions and runs as separate levels)
+%     tr (int): Repetition Time in seconds
+%   
+%     Returns: 
+%     design (sparse-double): sparse samples x stim-type matrices
+%     stimdur (int): stimulus duration in seconds
+%     ints2unique(dictionary): map between keys (integers of individual
+%       stimuli) and values (stimulus names)
+%     unique2ints(dictionary): self-explanatory
+
+% Initialize variables
+n = session_number; file_list = []; filename_list = {}; all_events={}; design = {}; stim_ids = char(); ints2unique={};
+fprintf('Transforming subject %d''s .tsv event files for "%s" sessions into design matrices.\n', sub, session_type)
+
+% Create path structures
+type_path = strcat(BIDS_path, 'sub-0', string(sub), '/ses-', session_type,'*');
```

Comment:
from my personal experience of switching often between windows and linux, you want to be careful to not write path with hardcoded `/` or `/`.

try using the `filesep` or `fullfile` function instead. :-)

Reply:
Good catch, fixed it! Thanks